OpenLIFULib/OpenLIFULib/lazyimport.py

```diff
@@ -18,7 +18,12 @@ def install_python_requirements() -> None:
 
 def python_requirements_exist() -> bool:
     """Check and return whether python requirements are installed."""
-    return importlib.util.find_spec('openlifu') is not None
+    try:
+        import threadpoolctl
+        import bcrypt
+    except ModuleNotFoundError:
+        return False
+    return importlib.util.find_spec('openlifu') is not None # openlifu import causes a delay so we check for it without actually importing yet
```

Comment:
we should have added this `try` long ago when bcrypt was added to `python-requirements.txt`.

so here I include bcrypt in addition to threadpoolctl